app/messages/Messages.scala

```diff
@@ -1769,9 +1764,59 @@ object Messages {
       }
     }
 
+    val `Call the debt management helpline`: Message = Message(
+      english = "Call the debt management helpline",
+      welsh   = "Ffoniwch y llinell gymorth rheoli dyledion"
+    )
+
+    val `Telephone...`: Message = Message(
+      english = "Telephone: <strong>0300 123 1813</strong>",
+      welsh   = "Ffôn: <strong>0300 200 1900</strong>"
+    )
+
+    val `Outside UK...`: Message = Message(
+      english = "Outside UK: <strong>+44 2890 538 192</strong>",
+      welsh   = "O’r tu allan i’r DU: +44 300 123 1813"
```

Comment:
does the welsh phone number need to be bold as well or is the difference with the english deliberate?

Reply:
This line is never shown in Welsh at all. I left something just in case, but I removed it now.